### DISCUSSION TITLE
Pre Search/index characters replace

### DISCUSSION THREAD

**Discussion by lenisko** [squidfunk/mkdocs-material#5905] [Ideas]
Would it be possible to replace characters before indexing and search?

As an example, I have Polish sentence: `Zażółć gęślą jaźń`. I would like to replace all Polish characters to ASCII and index it like `Zazolc gesla jazn` and do the same thing during search. Question is not limited to Polish characters.

**Comment by squidfunk** [comment]
That's known as stemming. We're using [lunr-languages](https://lunrjs.com/guides/language_support.html) for multi-language search support, which sadly, currently doesn't seem to support Polish (follow the link to see). However, we're in the process of rewriting the search from scratch, and will try to provide better support for [all languages that we support](https://squidfunk.github.io/mkdocs-material/setup/changing-the-language/#site-language). There's currently no ETA for that, as it's a pretty bug fish to fry.
